packages/cli/src/commands/create/monorepo.ts

```diff
@@ -99,6 +100,11 @@ export const createMonorepo: CreateMonorepoFunc = async ({
       return
     }
 
+    // skip yarn files if yarn is not the package manager
+    if (file.includes('.yarn') && packageManager !== 'yarn') {
+      return
+    }
```

Comment:
```suggestion
    if ((file === '.yarn' || file === '.yarnrc') && packageManager !== 'yarn') {
      return
    }
```

what if we go more specific? :)